js/Toolbar/Data-binding.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Data-binding
-description: data binding
+title: Data-binding in JavaScript Toolbar | Syncfusion
```

Comment:
And Control word in all titles if not added

Reply:
Updated